### ISSUE TITLE
Qt autocomplete

### ISSUE THREAD

**Issue by opensw** [microsoft/vscode-cpptools#666]
Hello there,
   for some reason I do not have any autocomplete on Qt types.

```
#include <qstring.h>

int main(int argc, char* argv[])
{
	QString str = "Hello world";
}
```

on str. Visual Studio Code does not show any methods, is there a workaround to fix it? The "Peek Definition" and "Go to Definition" commands work.

**Comment by sean-mcmanus** [comment]
I reproed the bug. Adding
`#define Q_DECL_NOTHROW`
`#define Q_REQUIRED_RESULT`
`#define Q_ATTRIBUTE_FORMAT_PRINTF(arg1, arg2)`
to cpp.hint fixes the issue (the file is located at the root of our extension folder so you could add it yourself if you want, and we'll update our version for the next release). Our parser code breaks when it encounters custom macro attributes at the end of a function declaration and uses #defines in cpp.hint to workaround it, so let us know if you find more we should add.

**Comment by opensw** [comment]
OK :)

Thank you

**Comment by bobbrow** [comment]
This is fixed in the Tag Parser in version 0.11.1. Please reopen if you continue to have this issue.
